transports/azure-service-bus-netstandard/index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Azure Service Bus .Net Standard Transport
+component: ASBS
+tags:
+ - Azure
+reviewed: 2018-06-19
+---
+
+## Azure Service Bus .Net Standard Transport
+
+The Azure Service Bus .Net Standard transport, ASBS in short, is the replacement of to the [Azure Service Bus transport](transports/azure-service-bus/) which leverages the .Net Standard compatible [Microsoft.Azure.ServiceBus](https://www.nuget.org/packages/Microsoft.Azure.ServiceBus/) client SDK.
```

Comment:
> The Azure Service Bus .Net Standard transport, ASBS in short, is the replacement

Please remove. We should simply state that ASBS is an Azure Service Bus transport that can run on .NET Core and full framework. Not a replacement. Not a successor. 

If I'm a ASB customer, reading successor or replacement would indicate that I'll get features from ASB in ASBS.